hugegraph-store/hg-store-core/pom.xml

```diff
@@ -121,9 +121,48 @@
             <version>${revision}</version>
             <scope>system</scope>
             <systemPath>
-                ${project.basedir}/../hg-store-dist/src/assembly/ext-lib/hugegraph-core-1.5.0.jar
+                ${top.level.dir}/hg-store-dist/src/assembly/ext-lib/hugegraph-core-1.5.0.jar
```

Comment:
add a TODO comment for it  

and unify the param
<img width="967" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/e9156218-c1e6-41fb-91d9-ab15f3c42e20">